src/main/java/com/financetracker/server/controller/CategoryController.java

```diff
@@ -33,20 +32,26 @@
         return categoryService.getAllCategoriesForUser();
     }
 
-    @PostMapping("/category/update")
-    public ResponseEntity<?> updateCategory(@RequestBody Category category) {
-        try{
-            categoryService.updateCategory(category);
+    @PostMapping("/category/{id}")
+    public ResponseEntity<?> updateCategory(@PathVariable long id, @Valid @RequestBody Category category) {
+        try {
+            categoryService.updateCategory(id, category);
```

Comment:
prečo posielame aj id? nie je to zbytočné?

Reply:
no podla mna to je takto dobre, tak som to robil vsade aj doteraz, ty posles v url, ktoru kategoriu chces updatnut (id), mozno dokonca ani nie post metodou ale put, aj tu je to takto popisane https://spring.io/guides/tutorials/rest/

Reply:
ok suhlasim teda

Reply:
upravis na put a mergnes?

Reply:
fixnute